src/Avalonia.Themes.Fluent/Controls/ToggleSwitch.xaml

```diff
@@ -106,13 +106,13 @@
                 CornerRadius="10" />
 
             <Canvas
-                x:Name="SwitchKnob"
+                x:Name="PART_SwitchKnob"
                 Grid.Row="1"
                 Width="20"
                 Height="20"
                 HorizontalAlignment="Left">
 
-              <Grid x:Name="MovingKnobs" Width="20" Height="20">
+              <Grid x:Name="PART_MovingKnobs" Width="20" Height="20">
```

Comment:
WinUI does not have two template parts in the ToggleSwitch like this (both "PART_SwitchKnob" and "PART_MovingKnobs"). Instead, there is only the one "SwitchKnob"

This may need to be simplified in Avalonia to match WinUI.

https://github.com/microsoft/microsoft-ui-xaml/blob/ea98f11e5748cdcdc22b153e175eef50adc7ad8c/dev/CommonStyles/ToggleSwitch_themeresources_v1.xaml#L546-L563